app/views/setup/_registry.html.slim

```diff
@@ -0,0 +1,15 @@
+= f.fields_for "insecure_registries[]", [] do |field|
+  .row.registry-entry
+    .form-group.col-sm-3
+      = field.text_field :docker_registry_name, placeholder: "Registry name", type: "text", value: (local_assigns[:registry] ? registry.name : "" ), disabled: !local_assigns[:registry].nil?
+    .form-group.col-sm-7
+      = field.text_area :docker_registry_certificate, placeholder: "Certificate", type: "text", value: (local_assigns[:registry] ? Certificate::DockerRegistry.find(registry.certificate_docker_registry_id).certificate : ""), cols: 80, rows: 20
```

Comment:
Can we put `Certificate::DockerRegistry.find(registry.certificate_docker_registry_id).certificate` in a variable/helper?